.github/workflows/tests.yml

```diff
@@ -61,6 +61,7 @@ jobs:
           # stumbles over. for some reason, mypy cannot be
           # instructed to ignore 'version.py'
           sed -i "s/# type:.*//" odxtools/version.py
+          python -m mypy --install-types
```

Comment:
Changes to CI can not be mixed to other changes such as adding a new feature

Reply:
Outdated, missing packages for CI test were added to pyproject.toml